### PR TITLE
test deployment + updated dependencies

### PR DESCRIPTION
**Pull request by wochinge**
- fix https://github.com/RasaHQ/rasa-x-helm/issues/27 (add test deployment to CI)
- fix https://github.com/RasaHQ/rasa-x-helm/issues/31 (update subcharts (the old ones were deprecated))
- replace linting in CI with `helm lint` since the linting GitHub action is not Helm 3 compatible

### CODE REVIEW

**Comment by wochinge** [comment]
I feel like I replaced everything under the hood :-D

File: charts/rasa-x/templates/nginx-deployment.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
     {{- include "rasa-x.labels" . | nindent 4 }}
     app.kubernetes.io/component: nginx
 spec:
-  replicas: {{ default 1 .Values.nginx.replicaCount }}
+  replicas: 1
```
Comment (mbelang) [inline charts/rasa-x/templates/nginx-deployment.yaml]: may I ask why this is now hardcoded to 1?

Comment (wochinge) [reply charts/rasa-x/templates/nginx-deployment.yaml]: Sure! Both components (rasa-x, nginx) don't make a lot of sense to scale at the moment. I can also re-add it, but I would still not recommend doing. Scaling nginx should not result in any problems though. Will re-add it 👍

Comment (mbelang) [reply charts/rasa-x/templates/nginx-deployment.yaml]: Yeah scaling nginx should not be a problem. For rasa-x you are probably right about rasa-x though

**Comment by wochinge** [comment]
I hope the release stuff still works 🤞

**Pull request merged**